wcomponents-core/src/main/java/com/github/bordertech/wcomponents/WMultiFileWidget.java

```diff
@@ -476,7 +488,9 @@ public void setThumbnailSize(final Dimension thumbnailSize) {
 						"Thumbnail size cannot have both height and width set to -1.");
 			}
 		}
-		getOrCreateComponentModel().thumbnailSize = thumbnailSize;
+		if (getThumbnailSize() == null || !thumbnailSize.equals(getThumbnailSize())) {
```

Comment:
Do we still make a component model if `getThumbnailSize() ==null` **and** the `thumbnailSize` arg is `null`?